app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -2081,6 +2081,7 @@ public void onClick(
             case R.id.refresh:
                 Log.w("menu", "Refreshed manually by user. We shouldn't need this.");
                 commonProgressDialog.showWithMessage(getString(R.string.refreshing));
+                commonProgressDialog.setCancelable(false);
```

Comment:
when I removed setCancelable() from the constructor and deployed the app on my phone, it had it's effect everywhere I added setCancelable after showWithMessage().